<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(148,169,216);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="grLampState">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>30</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold</string>
    </property>
    <property name="title">
     <string>Headlight Status</string>
    </property>
    <widget class="QPushButton" name="btnLampStateOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(40,40,40); color:white;</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampStateOn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(40,40,40); color:white;</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampOption">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(2,63,63); color:white;</string>
     </property>
     <property name="text">
      <string>MANUAL</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="lineItem1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>-180</y>
      <width>16</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpSignalStat">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>130</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold</string>
    </property>
    <property name="title">
     <string>Turn Signal Light Status</string>
    </property>
    <widget class="QPushButton" name="statTurnSignalL">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(40,40,40);  color:white;</string>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statSignalOff">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(40,40,40); color:white;</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statTurnSignalR">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(40,40,40); color:white;</string>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QLabel" name="statEmerSignal">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="device.qrc">:/images/system/emergency.png</pixmap>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpColorLampState">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>411</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:bold</string>
     </property>
     <property name="title">
      <string>Scooter Status LED</string>
     </property>
     <widget class="QPushButton" name="btnColorLedState">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="leColorLedValue">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpSensorState">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold</string>
    </property>
    <property name="title">
     <string>Sensor</string>
    </property>
    <widget class="QLineEdit" name="leTempValue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbTemperature">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(148,37,55);</string>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leCdsValue">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbIllumination">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(148,37,55);</string>
     </property>
     <property name="text">
      <string>CDS</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispDate">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2021/11/09</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispTime">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>11:12:13</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <zorder>leTempValue</zorder>
    <zorder>lbTemperature</zorder>
    <zorder>lbDispDate</zorder>
    <zorder>lbDispTime</zorder>
    <zorder>lbIllumination</zorder>
    <zorder>leCdsValue</zorder>
   </widget>
   <widget class="Line" name="lineItem2">
    <property name="geometry">
     <rect>
      <x>867</x>
      <y>10</y>
      <width>16</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSpeedValue">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSpeed">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCtrlAccelState">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>16</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbPerIllumin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHeadLight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCtrlBrakeState">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>16</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTurnSignalR1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTurnSignalL1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpDebugMsg">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>351</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="title">
     <string> Debug</string>
    </property>
    <widget class="QPushButton" name="btnDebugMsgClear">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>470</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="teDebugMessage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>301</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white; font-weight:normal;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbPMDevicePicture_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>-10</y>
      <width>371</width>
      <height>711</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>711</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>371</width>
      <height>711</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="device.qrc">:/images/system/front2.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTurnSignalR2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTurnSignalR3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTurnSignalL2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTurnSignalL3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSpeedTimer1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSpeedTimer2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSpeedTimer3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSpeedTimer4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(40,40,40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbSpeed_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight:bold;</string>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGear">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:grb(40,40,40); color:white; font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lbPMDevicePicture_2</zorder>
   <zorder>grLampState</zorder>
   <zorder>lineItem1</zorder>
   <zorder>grpSignalStat</zorder>
   <zorder>grpSensorState</zorder>
   <zorder>lineItem2</zorder>
   <zorder>lbSpeed</zorder>
   <zorder>pbCtrlAccelState</zorder>
   <zorder>lbPerIllumin</zorder>
   <zorder>pbHeadLight</zorder>
   <zorder>pbCtrlBrakeState</zorder>
   <zorder>pbTurnSignalR1</zorder>
   <zorder>pbTurnSignalL1</zorder>
   <zorder>grpDebugMsg</zorder>
   <zorder>leSpeedValue</zorder>
   <zorder>pbTurnSignalR2</zorder>
   <zorder>pbTurnSignalR3</zorder>
   <zorder>pbTurnSignalL2</zorder>
   <zorder>pbTurnSignalL3</zorder>
   <zorder>pbSpeedTimer1</zorder>
   <zorder>pbSpeedTimer2</zorder>
   <zorder>pbSpeedTimer3</zorder>
   <zorder>pbSpeedTimer4</zorder>
   <zorder>lbSpeed_2</zorder>
   <zorder>pbGear</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="device.qrc"/>
 </resources>
 <connections/>
</ui>
